tool/ci/cmd/bot.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/gravitational/teleport/tool/ci"
+	"github.com/gravitational/teleport/tool/ci/pkg/bot"
+	bots "github.com/gravitational/teleport/tool/ci/pkg/bot"
+	"github.com/gravitational/teleport/tool/ci/pkg/environment"
+	"github.com/gravitational/trace"
+
+	"github.com/google/go-github/v37/github"
+	"golang.org/x/oauth2"
+)
+
+func main() {
+	var token = flag.String("token", "", "token is the Github authentication token.")
+	var reviewers = flag.String("reviewers", "", "reviewers is a string representing a json object that maps authors to required reviewers for that author.")
+	flag.Parse()
+
+	subcommand := os.Args[len(os.Args)-1]
+	ctx := context.Background()
```

Comment:
Might be good to use a context with a timeout to prevent a run from hanging for some reason and consuming a bunch of GitHub actions time.